tests/integration/testlib/statelib.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+from libnmstate import netinfo
+
+
+INTERFACES = 'interfaces'
```

Comment:
Shouldn't this constant be part of libnmstate?

Reply:
I guess it should be promoted there. A PR on that will be nice.
(where to locate it is less obvious I guess)

Reply:
See #110

Reply:
Thank you